src/app/components/BibPage/BibMainInfo.jsx

```diff
@@ -1,34 +1,75 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router';
 import { isEmpty as _isEmpty } from 'underscore';
 
 const BibMainInfo = ({ bib = {} }) => {
   if (_isEmpty(bib)) return null;
 
-  const title = bib.title && bib.title.length ? bib.title[0] : '';
   const materialType = bib && bib.materialType && bib.materialType[0] ?
     bib.materialType[0].prefLabel : null;
+  const author = bib.creatorLiteral && bib.creatorLiteral.length ?
+    `${bib.creatorLiteral[0]}` : '';
+  const contributor = bib.contributorLiteral && bib.contributorLiteral.length ?
+    bib.contributorLiteral : null;
+  const yearPublished = bib && bib.dateStartYear ? bib.dateStartYear : null;
+  const placeOfPublication = bib && bib.placeOfPublication ? bib.placeOfPublication : null;
+  const publisher = bib && bib.publisher ? bib.publisher : null;
   const language = bib && bib.language && bib.language[0] ?
     bib.language[0].prefLabel : null;
-  const yearPublished = bib && bib.dateStartYear ? bib.dateStartYear : null;
-  const usageType = bib && bib.actionType && bib.actionType[0] ?
-    bib.actionType[0].prefLabel : null;
 
   return (
-    <div className="nypl-item-details">
-      <h1>{title}</h1>
-      <div className="nypl-item-info">
-        <p>
-          <span className="nypl-item-media">{materialType}</span>
-          {language && ` in ${language}`}
-        </p>
-        <p>{bib.extent} {bib.dimensions}</p>
-        <p>
-          {bib.placeOfPublication} {bib.publisher} {yearPublished}
-        </p>
-        <p className="nypl-item-use">{usageType}</p>
-      </div>
-    </div>
+    <dl>
+      <dt>FORMAT</dt>
+      <dd>
+        {materialType}
+        {language && ` in ${language}`}
+      </dd>
+      <dt>AUTHOR</dt>
+      <dd>
+        <Link
+          onClick={e => this.newSearch(e, author)}
+          title={`Make a new search for: ${author}`}
+          to={`/search?${author}`}
+        >
+          {author}
+        </Link>
+      </dd>
+      <dt>CONTRIBUTORS</dt>
+      <dd>
+        {
+          contributor ? contributor.map((valueObj, i) => {
```

Comment:
We can do this(Learned from Edwin), so you don't need to put `null` in the end

`
{
          contributor && contributor.map((valueObj, i) => {
           return (
              <span key={i}>
               <Link
                  onClick={e => this.newSearch(e, valueObj)}
                 title={`Make a new search for: ${valueObj}`}
                 to={`/search?${valueObj}`}
                >
                  {valueObj}
                </Link>,&nbsp;
             </span>
            );
         }) 
       }
`